### PR TITLE
Add travis integration

### PR DESCRIPTION
**Pull request by scholzj**
This PR adds Travis integration.

### CODE REVIEW

File: .travis/build.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+# Push only releases
+if [ "$PULL_REQUEST" != "false" ] ; then
+    echo "Building Pull Request - nothing to push"
+elif [ "$TAG" = "latest" ] && [ "$BRANCH" != "master" ]; then
+    echo "Not in master branch and not in release tag - nothing to push"
+else
+    if [ "${MAIN_BUILD}" = "TRUE" ] ; then
+        echo "Pushing JARs"
+        ./.travis/push-to-nexus.sh
```
Comment (ppatierno) [inline .travis/build.sh]: Is the entire path needed here?

File: pom.xml [hunk 2]
```diff
@@ -8,6 +8,51 @@
...
+            <name>Apache License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <name>Strimzi - Apache Kafka on Kubernetes and OpenShift</name>
+    <description>Strimzi uses the Kubernetes operator pattern to provide a way to run an Apache Kafka cluster on
+        Kubernetes or OpenShift in various deployment configurations. The OAuth subproject implements the callbacks for
+        Kafka authentication using OAuth (based on SASL OAUTHBEARER mechanism).</description>
+    <url>https://strimzi.io/</url>
```
Comment (ppatierno) [inline pom.xml]: Should we remove OpenShift mentions here?

File: .travis/build.sh (hunk 1 above)
Comment (scholzj) [reply .travis/build.sh]: I'm quite sure I added it for a reason. But it was long time ago (here just copied from our main repo). Do you want me to try it?

File: pom.xml (hunk 2 above)
Comment (scholzj) [reply pom.xml]: Done. This is copied from the main operators repo. We should maybe remove it there as well.

File: .travis/build.sh (hunk 1 above)
Comment (ppatierno) [reply .travis/build.sh]: No need to change then ;)

File: pom.xml (hunk 2 above)
Comment (ppatierno) [reply pom.xml]: Yes agree

**Pull request merged**